lib/cucumber/cli/main.rb

```diff
@@ -14,7 +14,12 @@ def execute(args)
         end
       end
 
-      def initialize(args, _ = nil, out = STDOUT, err = STDERR, kernel = Kernel)
+      # The second argument is there for retrocompatibility
```

Comment:
Not sure on how things work here, but given it's an unused argument I assume we plan on removing this in cucumber v8 ruby?

Reply:
I am pro removing it indeed.
And as it is unused, maybe we could consider it was already deprecated, thus removing it in v8